spec/03-plugins/18-acl/01-api_spec.lua

```diff
@@ -152,10 +153,114 @@ for _, strategy in helpers.each_strategy() do
           res = assert(admin_client:get("/consumers/alice/acls/" .. acl.group))
           assert.res_status(404, res)
         end)
+        it("retrieves right ACL by group when multiple consumers share the same group name created with POST", function()
+          local res  = assert(admin_client:post("/consumers", {
```

Comment:
All the "shortcut request methods" (`admin_client:post`, `admin_client:delete`, `admin_client:get`, etc) already do an `assert`, so wrapping them with an `assert` is not really needed.